packages/shared/src/focus-visible.ts

```diff
@@ -40,6 +38,9 @@ try {
     document.body.querySelector(':focus-visible');
 } catch (error) {
     hasFocusVisible = false;
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    import('focus-visible');
```

Comment:
This is the guts of this PR. Everything else is test support for this change in browsers that do not require the polyfill, which is ALL of them now! 🥳